### ISSUE TITLE
Nullpointer when trying to parse manifest file

### ISSUE THREAD

**Issue by muhammedashraf9244** [wt-io-it/odoo-pycharm-plugin#116]
Plugin Version: 0.6.9
IntelliJ Version: PyCharm 2022.3.1
IntelliJ Build: PC-223.8214.51

<details><summary>java.lang.NullPointerException: Cannot invoke "com.intellij.psi.PsiFile.getProject()" because "manifestFile" is null</summary>

```
java.lang.NullPointerException: Cannot invoke "com.intellij.psi.PsiFile.getProject()" because "manifestFile" is null
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooManifestParser.parse(OdooManifestParser.java:26)
	at at.wtioit.intellij.plugins.odoo.modules.index.OdooDeserializedModuleImpl.getDependencies(OdooDeserializedModuleImpl.java:83)
	at at.wtioit.intellij.plugins.odoo.modules.impl.AbstractOdooModuleImpl.dependsOn(AbstractOdooModuleImpl.java:62)
	at java.base/java.util.stream.MatchOps$1MatchSink.accept(MatchOps.java:90)
	...
	at java.base/java.util.stream.ReferencePipeline.noneMatch(ReferencePipeline.java:642)
	at at.wtioit.intellij.plugins.odoo.models.impl.OdooModelImpl.lambda$getBaseModule$11(OdooModelImpl.java:157)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:178)
	...
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:647)
	at at.wtioit.intellij.plugins.odoo.models.impl.OdooModelImpl.lambda$getBaseModule$12(OdooModelImpl.java:158)
	at at.wtioit.intellij.plugins.odoo.WithinProject.call(WithinProject.java:12)
	at at.wtioit.intellij.plugins.odoo.models.impl.OdooModelImpl.getBaseModule(OdooModelImpl.java:150)
	at at.wtioit.intellij.plugins.odoo.models.impl.OdooModelImpl.lambda$getDefiningElement$2(OdooModelImpl.java:60)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:178)
	...
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
	at at.wtioit.intellij.plugins.odoo.models.impl.OdooModelImpl.lambda$getDefiningElement$5(OdooModelImpl.java:63)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:860)
	at at.wtioit.intellij.plugins.odoo.models.impl.OdooModelImpl.getDefiningElement(OdooModelImpl.java:49)
	at at.wtioit.intellij.plugins.odoo.records.index.OdooDeserializedRecordImpl.getDefiningElement(OdooDeserializedRecordImpl.java:55)
	at at.wtioit.intellij.plugins.odoo.AbstractOdooCompletionContributor.lambda$suggestRecordXmlId$1(AbstractOdooCompletionContributor.java:119)
	at at.wtioit.intellij.plugins.odoo.WithinProject.run(WithinProject.java:25)
	at at.wtioit.intellij.plugins.odoo.AbstractOdooCompletionContributor.suggestRecordXmlId(AbstractOdooCompletionContributor.java:118)
	at at.wtioit.intellij.plugins.odoo.OdooCompletionContributor.fillCompletionVariants(OdooCompletionContributor.java:52)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributor(CompletionService.java:81)
	...
	at java.base/java.lang.Thread.run(Thread.java:833)
```
</details>

**Comment by bigbear3001** [comment]
this is fixed with 0.6.11 (pending jetbrains approval) if you are eager to test it you can use the jar file from: https://github.com/wt-io-it/odoo-pycharm-plugin/tree/v0.6.11

same as #115

**Comment by ap-wtioit** [comment]
0.6.11 is now available in the marketplace
